src/libraries/System.IO.Compression/src/System/IO/Compression/ZipCompressionMethod.cs

```diff
@@ -0,0 +1,29 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.IO.Compression
+{
+    /// <summary>
+    /// Specifies the compression method used to compress an entry in a zip archive.
+    /// </summary>
+    /// <remarks>
+    /// The values correspond to the compression method values described in the ZIP File Format Specification (APPNOTE.TXT section 4.4.5).
+    /// </remarks>
+    public enum ZipCompressionMethod : short
```

Comment:
The integer is unsigned in the approved proposal.

```suggestion
    public enum ZipCompressionMethod : ushort
```